File: en-game/back-end/admin-service/src/main/java/com/endgame/adminservice/dto/category/CategoryExportExcelDTO.java
package com.endgame.adminservice.dto.category;

import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@Getter
@Setter
@Data
public class CategoryExportExcelDTO {
    private int id;
    private int parentId;
    private String name;
    private boolean active;
    private boolean hasSubCategory;
}

File: en-game/front-end-admin/src/app/main/dto/Category/CategoryExportExcelDTO.ts
export class CategoryExportExcelDTO {
    index: number;
    id: number;
    parentId: number;
    name: string;
    active: boolean;
    hasSubCategory: boolean;

    constructor(index: number, id: number, parentId: number, name: string, active: boolean, hasSubCategory: boolean){
        this.index = index;
        this.id = id;
        this.parentId = parentId;
        this.name = name;
        this.active = active;
        this.hasSubCategory = hasSubCategory;
    }
}